packages/react-components/react-drawer/stories/src/Drawer/InlineDrawer.stories.tsx

```diff
@@ -58,29 +58,18 @@ type DrawerInlineExampleProps = DrawerProps & {
   setOpen: (open: boolean) => void;
 };
 
-const getMappedPosition = (position: DrawerProps['position']) => {
-  switch (position) {
-    case 'end':
-      return 'Right';
-
-    case 'bottom':
-      return 'Bottom';
-
-    default:
-      return 'Left';
-  }
-};
+const capitalize = (str?: string) => str && str.charAt(0).toUpperCase() + str.slice(1);
 
 const setButtonText = (open: boolean, position: DrawerProps['position']) => {
   let buttonText = open ? 'Close' : 'Open';
 
   switch (position) {
     case 'start':
-      buttonText = `${buttonText} left`;
+      buttonText = `${buttonText} start`;
```

Comment:
switch case for this looks a bit verbose, before it was start -> left, now I think you can something like:

```js
... 
  if (position === 'start' || position === 'end' || position === 'bottom') {
    return `${buttonText} ${position}`;
  }
 
return `${buttonText} drawer`
```